src/components/note/note.js

```diff
@@ -24,6 +24,10 @@ class Note extends React.Component {
       error: {
         background: '#fbe5e5',
         color: '#ba3b3f'
+      },
+      new: {
+        background: '#e8f5ee',
+        color: '#269561'
```

Comment:
Recommend a slightly darker text color to improve the color contrast: https://webaim.org/resources/contrastchecker/?fcolor=269561&bcolor=E8F5EE

```suggestion
        color: '#1b7d4f'
```

Color contrast with suggestion: https://webaim.org/resources/contrastchecker/?fcolor=1B7D4F&bcolor=E8F5EE

Reply:
Good thinking! I need to get better about remembering to use the contrast checker.